instrument/D7_YIG_calibration.xml

```diff
@@ -0,0 +1,432 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I don't understand why this file ended up here? We do need the YIG peaks, but this seems to be just a calibration output. Was there a requirement to have a default calibration or something?

Reply:
This calibration file is used for the unit test of the `LoadILLPolarizedDiffraction` algorithm. It has been moved from the UnitTests data folder.